handbook/documentation/index.md

```diff
@@ -12,7 +12,10 @@ We use the [monthly release blog post](https://about.sourcegraph.com/blog) as a
 
 ## Naming and linking documentation pages
 
-1. There is no global index or nav, so all docs should be linked from their parent index page. Every new document should be cross-linked to its related documentation, and linked from its topic-related index, when it exists.
+1. All docs should be linked to from their parent index page.
```

Comment:
What exactly do you mean by this? Every page in the current directory should be linked to in the first `index.md` in the directory hierarchy?

Reply:
I've changed this to plain language largely inspired by your question.